src/main/java/pwr/sim/animal/Animal.java

```diff
@@ -27,16 +27,16 @@ public void approach(Position2D dest) {
         int y = diff.getY();
         int stepX = 0, stepY = 0;
         if(x < 0) {
-            stepX = -1;
-        } else if(x > 0) {
             stepX = 1;
+        } else if(x > 0) {
```

Comment:
Just so I'm sure I'm understanding correctly: the step directions here are reversed because if an animal's target is on its right side, then `position.delta(dest)` returns `Position2D` with `x < 0`? The cleaner fix would be to change `position.delta(dest)` to `dest.delta(position)` but gonna merge it either way